src/rust/catnap/linux/transport.rs

```diff
@@ -38,7 +38,7 @@ use crate::{
         DemiRuntime,
         SharedDemiRuntime,
         SharedObject,
-    },
+    }, timer,
```

Comment:
Not sure if this was auto-formatted (if it was, just ignore).
To keep consistency, could you move this to the next line?

Reply:
Yes, I will move it. Thanks.